src/configurations/destinations/http/ui-config.json

```diff
@@ -457,6 +457,22 @@
           {
             "name": "Request Body",
             "fields": [
+              {
+                "type": "textInput",
+                "label": "XML Root Key",
```

Comment:
Add note with "Enter the root key to be used as a common prefix for all field mappings being defined below"

Reply:
Updated